src/Playwright/Transport/StdIOTransport.cs

```diff
@@ -153,60 +127,37 @@ private void Dispose(bool disposing)
             _process?.Dispose();
         }
 
-        private async Task GetResponseAsync(CancellationToken token)
+        private async Task OnMessageAsync(CancellationToken token)
         {
-            try
-            {
-                var stream = _process.StandardOutput;
-                byte[] buffer = new byte[DefaultBufferSize];
+            var stream = _process.StandardOutput;
```

Comment:
Maybe get the `BaseStream` here so `.BaseStream` isn't needed below.

```suggestion
            var stream = _process.StandardOutput.BaseStream;
```